### PR TITLE
Add contact form

### PR DESCRIPTION
**Pull request by Khawanin**
## Portfolio Mobile Version
 In this milestone we: 
- added contact form to the portfolio project
- used flexbox to align form elements
- the form accomplishment was done by my coding partner and me

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -206,6 +206,20 @@ <h2>Skills</h2>
 
     </div>
   </section>
+
+  <section class="form-background">
+    <h1 class="form-heading">Contact me</h1>
```
Comment (ShadyShawkat) [inline index.html]: - [ ] Please make sure that there's only one `h1` tag on your page and then the following header would follow a hierarchy. This is considered good practice and improves the SEO of your website.

File: index.html [hunk 2]
```diff
@@ -206,6 +206,20 @@ <h2>Skills</h2>
...
+    <p class="form-content">If you have an application you are interested in developing, a feature that you need built
+      or a project that needs coding. I’d love to help with it.</p>
+    <form class="contact-form" action="https://formspree.io/f/mvolvljr" method="POST">
+      <input class="name" type="text" name="firstname" maxlength="30" placeholder="Please enter your name " required>
+      <input class="email" type="email" name="email" placeholder="Please enter your email" required>
+      <textarea class="message" name="message" maxlength="500" placeholder="Write your message here"
+        required></textarea>
+      <button class="submit" type="submit">Get it touch</button>
+    </form>
+  </section>
```
Comment (ShadyShawkat) [inline index.html]: - [ ] Kindly remove the extra white space to the left and right of your design to match the Figma template.

image

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @Khawanin, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Khawanin** [comment]
@ShadyShawkat  
Hi, hope you are doing well.
I just did the changes as requested. But when I go to my 'progress' section in my dashboard, I can't see the 'request review' button. It looks like my pull request is approved and I have to share the link to my project only.

regards.

**Pull request merged**